services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -2054,6 +2054,8 @@ private Pair<Boolean, Version> addVersion(
           }
 
           topicToCreationTime.computeIfAbsent(version.kafkaTopicName(), topic -> System.currentTimeMillis());
+          // TODO: Once concurrent batch pushes handling no longer relies on topics on parent Kafka cluster, stop
```

Comment:
Another way is to leveraging HB system store to decide whether there are any concurrent pushes or not.

Reply:
Even without it, can Parent Controller just query all the child controller to decide whether there is any concurrent push or not?